src/app/workflow/workflow.component.html

```diff
@@ -88,6 +88,7 @@ <h3>
           [entryType]="entryType"
           [workflow]="workflow"
           [selectedVersion]="selectedVersion"
+          [zenodoAccountIsLinked]="zenodoAccountIsLinked"
```

Comment:
There should be nothing modified in both workflow.component.html and workflow.component.ts since they're not really related to the DOI button at all

Reply:
removed changes and refactored code based on your other comments